react/src/utils/backendrouter.js

```diff
@@ -1,16 +1,17 @@
 const DEFAULT_BACKEND = "flask"
+
 const SUPPORTED_BACKEND_TYPES = {
   "flask": {
       "test": "http://localhost:8080",
       "production": process.env.FLASK_BACKEND || process.env.REACT_APP_BACKEND
```

Comment:
`FLASK_BACKEND` should be prefixed with `REACT_APP_` (or just deleted, alternatively, since we already have REACT_APP_BACKEND...)

Reply:
good call